keyboards/annepro2/annepro2.c

```diff
@@ -26,53 +25,29 @@ uint16_t annepro2LedMatrix[MATRIX_ROWS * MATRIX_COLS] = {
 };
 
 void OVERRIDE keyboard_pre_init_kb(void) {
+#if HAL_USE_SPI == TRUE
+    spi_init();
+#endif
 }
 
 void OVERRIDE keyboard_post_init_kb(void) {
     // Start LED UART
     sdStart(&SD0, &ledUartConfig);
     sdWrite(&SD0, ledMcuWakeup, 11);
 
+    wait_ms(75);
```

Comment:
Probably add a comment explain why we need this wait

Reply:
Similar, missed this in the cherry pick, it's a delay for the eeprom to stabilize. I'll remove it.